actionpack/test/abstract_unit.rb

```diff
@@ -16,7 +16,7 @@
 if ENV["TRAVIS"]
```

Comment:
Not directly related to this pull request, I'm wondering if we can replace `ENV["TRAVIS"]` with `ENV["CI"]`.

https://github.com/rails/rails/commit/7b7b12f5983fd0c48e4a8dad909d3388b72f046c does not explain the reason why then I'm not sure if we can replace `ENV["TRAVIS"]` with ENV["CI"] yet.

Reply:
After replaced travis to buildkite, `ENV["TRAVIS"]` seems no longer works.